custom/icds_reports/utils/topojson_util/topojson_util.py

```diff
@@ -63,13 +63,39 @@ def copy_custom_metadata(from_topojson, to_topojson):
         location_data['scale'] = from_topojson['objects'][location_name]['scale']
 
 
-def get_topojson_for_district(district):
+def get_topojson_for_district(state, district):
```

Comment:
quick question for my knowledge. When one district is drawn on view. District file is used or blocks file?

Reply:
when you are zoomed into a single district, the block file is used (so that you can see the blocks within that district). Does that answer the question?